app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,17 @@
+class FeedbacksController < ApplicationController
+  expose(:feedback)
+
+  def new
+  end
+
+  def create
+    ApplicationMailer.feedback(feedback).deliver_now! if feedback.save
```

Comment:
For using `deliver_later` feedback should have id parameter. Should I add module `GlobalID::Identification`and add id and than find then object by id like this?

```
class Feedback
  include GlobalID::Identification
  
  def id 
    ...   
  end    

  def self.find(id)
    ...   
  end 
end